src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,9 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    Machine[] machine = new Machine[]{new Truck(), new Bulldozer(), new Excavator()};
```

Comment:
The array declaration is correct, but the loop that follows is not properly enclosed within a method. In Java, code must be inside a method or constructor.